### PR TITLE
Add list of orders and promo summary

### PR DESCRIPTION
**Pull request by jelneo**
1. Display a list of all orders ordered by the most recent order time
 -- Also displays the food items ordered per order
2. Show promotion summary

### CODE REVIEW

File: controllers/staff.js [hunk 1]
```diff
@@ -50,6 +50,37 @@ LIMIT $4;
...
+),
+RidToName AS (
+    SELECT uid as riderId, name as rname
+    FROM Riders natural join Users
+),
+CidToName AS (
+    SELECT uid as customerId, name as cname
+    FROM Customers natural join Users
+)
+SELECT oid, rname, cname, orderTime, deliveredTime, finalPrice, (SELECT string_agg(fname, ', ') FROM COLLATES C WHERE C.oid = X.oid) as itemsOrdered
```
Comment (MuhdNurKamal) [inline controllers/staff.js]: correct me if I'm wrong, the food is returned as a comma seperated string right? I dont think this is good. What if you have a food entry named: "Egg, Chicken and rice", the front end doesnt know whether theres 2 types of food or 1 type of food

File: controllers/staff.js [hunk 2]
```diff
@@ -50,6 +50,37 @@ LIMIT $4;
...
+FROM (Orders natural join OrderRestaurantPair join RidToName using (riderId) join CidToName using (customerId)) as X
```
Comment (MuhdNurKamal) [inline controllers/staff.js:80]: What's X?

Comment (jelneo) [reply controllers/staff.js:80]: table alias. it is used in the line above to qualify oid

File: controllers/staff.js (hunk 1 above)
Comment (jelneo) [reply controllers/staff.js]: ok ill separate it with `|` instead

Comment (MuhdNurKamal) [reply controllers/staff.js]: And if the name cotains | ?

Comment (MuhdNurKamal) [reply controllers/staff.js]: Please use array_agg(fname) instead

**Pull request merged**